### PR TITLE
Remove gRPC proto compiler invocation from CMakeLists.txt

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -115,20 +113,16 @@ if(${BUILD_GRPC})
 
     list(APPEND ALL_GRPC_SERVICE_SRCS "${PROJECT_BINARY_DIR}/${DIR}/${FIL_WE}.pb.cc")
     list(APPEND ALL_GRPC_SERVICE_HDRS "${PROJECT_BINARY_DIR}/${DIR}/${FIL_WE}.pb.h")
-    list(APPEND ALL_GRPC_SERVICE_SRCS "${PROJECT_BINARY_DIR}/${DIR}/${FIL_WE}.grpc.pb.cc")
-    list(APPEND ALL_GRPC_SERVICE_HDRS "${PROJECT_BINARY_DIR}/${DIR}/${FIL_WE}.grpc.pb.h")
 
     add_custom_command(
       OUTPUT "${PROJECT_BINARY_DIR}/${DIR}/${FIL_WE}.grpc.pb.cc"
```
Comment (drigz) [inline CMakeLists.txt]: Above you remove `${FIL_WE}.grpc.pb.*` but here they remain - is that intended?

Comment (cschuet) [reply CMakeLists.txt]: Thanks for catching that.

**Comment by cschuet** [comment]
@drigz exactly :)

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**